pkg/templates/images/images.go

```diff
@@ -355,7 +355,9 @@ func optionalResources() map[Resource]map[string]string {
 		// vSphere CCM
 		VsphereCCM: {
 			"1.24.x":    "gcr.io/cloud-provider-vsphere/cpi/release/manager:v1.24.2",
-			">= 1.25.0": "gcr.io/cloud-provider-vsphere/cpi/release/manager:v1.25.0",
+			"1.25.x":    "gcr.io/cloud-provider-vsphere/cpi/release/manager:v1.25.0",
+			"1.26.x":    "gcr.io/cloud-provider-vsphere/cpi/release/manager:v1.26.0",
```

Comment:
As per https://github.com/kubernetes/cloud-provider-vsphere, there are newer patch releases available. Can we please upgrade to those newer patch releases?